src/entity/Option.ts

```diff
@@ -26,4 +33,11 @@ export class Option {
     { onDelete: 'CASCADE' },
   )
   room: Room;
+
+  static New({ label, value }) {
```

Comment:
`Option.New` doesn't look to idiomatic, what about `Option.build`?
```suggestion
  static build({ label, value }) {
```

Reply:
It seems a good idea for me uppercasing static methods since they are part of the "class" definition and we usually uppercase class names, although I agree is not too idiomatic.   

On the other hand ```build``` feels weird outside the builder pattern.  
What about ```new``` or ```create``` ?

Reply:
I'm not familiar with the builder pattern apparently, but for me `build` function can either return attributes or a built instance depending on the context. For some reason `new`  doesn't look appealing to me. Since `Option` is also a typeorm entity I think adding a `create` may be confusing